packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -57,16 +57,15 @@ export const CloudPulseRegionSelect = React.memo(
         (item: CloudPulseResourceTypeMapFlag) =>
           item.serviceType === serviceType
       );
-
       const supportedRegionsIdList =
         resourceTypeFlag?.supportedRegionIds
           ?.split(',')
-          .map((regionId: string) => regionId.trim()) || [];
+          .map((regionId: string) => regionId.trim())
+          .filter((regionId: string) => regionId.length > 0) || [];
```

Comment:
const supportedRegionsIdList = (resourceTypeFlag && resourceTypeFlag.supportedRegionIds) ? 
    resourceTypeFlag.supportedRegionIds
      .split(',')
      .map((regionId: string) => regionId.trim())
      .filter((regionId: string) => regionId.length > 0) : []'``

Reply:
@ankita-akamai , does this looks more understandable